js/gal.js

```diff
@@ -1,15 +1,24 @@
 // Welcome to UK Postcode Planner .js file 
 
-// Postcode API base endpoint and specific endpoints for varying methods. E.g. look up postcode, random postcode etc.    
+// Postcode API base endpoint and specific endpoints for varying methods. E.g. look up postcode, random postcode etc.
+
 const baseEndpoint = 'https://api.postcodes.io';
 const randomPostcodeEndpoint = `${baseEndpoint}/random/postcodes`
+const uniquePostcodeEndpoint = `${baseEndpoint}/postcodes`
```

Comment:
awesome that you attracted `baseEndpoint` into its own variable

Reply:
Thanks 🙏